decidim-design/app/helpers/decidim/design/announcement_helper.rb

```diff
@@ -63,7 +63,7 @@ def announcement_sections
                   cell: "decidim/announcement",
                   args: [{ title: "This is the title", body: "This is the body" }, { callout_class: "success" }],
                   call_string: [
-                    'cell("decidim/announcement", { title: "This is the title", body: "This is the body" }, callout_class: "success")',
+                    %{cell("decidim/announcement", { title: "#{t("decidim.design.helpers.send")}", body: "This is the body" }, callout_class: "success")},
```

Comment:
I am not convinced this change needs to be here. 
```suggestion
                    %{cell("decidim/announcement", { title: "This is the title", body: "This is the body" }, callout_class: "success")},
```

Reply:
So don't change strings in all cells? But if argument is being translated, doesn't that need to be consistent also? 

Same goes for the changes I did in PR: https://github.com/decidim/decidim/pull/12573/files